src/core/utils/KeyMapping.ts

```diff
@@ -0,0 +1,62 @@
+import { defaultKeyMap } from './DefaultKeyMap';
+
+export interface KeyModifiers {
+    ctrlKey?: boolean;
+    altKey?: boolean;
+    shiftKey?: boolean;
+}
+export interface Keypress {
+    key: string;
+    modifiers: KeyModifiers;
+}
+export interface IntentInformation {
+    name: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    arguments: Array<any> | Record<string, any>;
```

Comment:
Nicolas' PR will soon replace Intents by Commands. I guess this will become `CommandConfig` or something like that when altered in his branch ? I'm thinking about this because my brain can't seem to get what `IntentInformation` is supposed to mean but I'm not sure it makes a lot of sense to spend time into finding a better name while Intents will soon ceast to exist in their current form.